libnetwork/osl/sandbox.go

```diff
@@ -17,6 +17,12 @@ const (
 	SandboxTypeLoadBalancer = iota
 )
 
+// IfaceOption is a function option type to set interface options.
+type IfaceOption func(i *nwIface)
+
+// NeighOption is a function option type to set neighbor options.
+type NeighOption func(nh *neigh)
```

Comment:
TBH; we should look at these as they use a non-exported type as part of their type, which effectively means they cannot be implemented outside of this package.

They're currently part of an interface, but perhaps we don't need that if we move code to be with the correct build-tags (I kept that for a later exercise)